frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -134,6 +134,19 @@ const plugin: Plugin<ConsumedExtensions> = [
         ),
     },
   },
+  {
+    type: 'Page/Route',
+    properties: {
+      exact: true,
+      path: `/k8s/ns/:ns/${referenceForModel(
+        ClusterServiceVersionModel,
+      )}/:appName/${apiObjectRef}/~new`,
+      loader: () =>
+        import('./components/ocs-install/install-page' /* webpackChunkName: "install-page" */).then(
+          (m) => m.default,
```

Comment:
Shouldn't the above `'Page/Route` should be updated with these details? Why another object is required?

Reply:
We need to support both.